digital_image_processing/edge_detection/canny.py

```diff
@@ -110,6 +115,24 @@ def canny(image, threshold_low=15, threshold_high=30, weak=128, strong=255):
                 else:
                     dst[row, col] = 0
 
+
+def canny(image, threshold_low=15, threshold_high=30, weak=128, strong=255):
+    image_row, image_col = image.shape[0], image.shape[1]
```

Comment:
```suggestion
    image_row, image_col = image.shape
```

Reply:
This function is not using `image.shage` so why bother unpacking it?

Make all the functions use image.shape instead of separate row and col.

Reply:
Done